tests/unit/models/test_base_model.py

```diff
@@ -388,6 +418,83 @@ def test_from_dict_submit(self):
         actual = Request.from_dict(request)
         self.assertEqual(actual, expected)
 
+    def test_nonexistent_field(self):
+        tx = {
+            "account": "rH6ZiHU1PGamME2LvVTxrgvfjQpppWKGmr",
+            "bad_field": "random",
+            "flags": 131072,
+            "limit_amount": {
+                "currency": "USD",
+                "issuer": "raoV5dkC66XvGWjSzUhCUuuGM3YFTitMxT",
+                "value": "100",
+            },
+        }
+        with self.assertRaises(XRPLModelException):
+            TrustSet.from_dict(tx)
+
+    def test_bad_literal(self):
+        tx = {
```

Comment:
could you add a comment and point out the `bad_literal` attribute in this test case?